src/status_im/ui/screens/events.cljs

```diff
@@ -144,6 +144,13 @@
     (fn [encryption-key]
       (re-frame/dispatch [:initialize-app encryption-key])))))
 
+(re-frame/reg-fx
+ ::get-encryption-key-fx
```

Comment:
I can't see any usage of this, ( I have found `::got-encryption-key-fx` in develop, but no `::get-encryption-key-fx` in this PR nor develop), what am I missing?

Reply:
@yenda did you have a chance to look at this?

Reply:
@cammellos yes something was wrong with my commit a bunch of changes went missing but then I had to rework the PR anyway to account for rinkeby